lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -9,9 +9,11 @@ import RxDataSources
 import Storage
 import Shared
 
-protocol ItemListViewProtocol: class, AlertControllerView {
+protocol ItemListViewProtocol: class, AlertControllerView, SpinnerAlertView {
```

Comment:
This makes sense to have `SpinnerAlertView` here, but only if `UIViewController` isn't extended by `SpinnerAlertView`.